packages/backend/src/database/seeds/development/02_saved_queries.ts

```diff
@@ -9,6 +9,7 @@ export async function seed(knex: Knex): Promise<void> {
     // Inserts seed entries
     await savedChartModel.create(SEED_PROJECT.project_uuid, {
         name: 'How much revenue do we have per payment method?',
+        description: 'How much revenue do we have per payment method?',
```

Comment:
Total revenue received via coupons, gift cards, bank transfers, and credit cards